### ISSUE TITLE
QM authority

### ISSUE THREAD

**Issue by scottpeterson** [gryphonshafer/Quizzing-Rule-Book#46]
State clearly that QMs have quiz question ruling authority over AJs.

**Comment by kclimenhaga** [comment]
I agree that defining the chain of decision making is probably a good idea. I am not sure whether the quizmaster should be the one with final say, but maybe that's because I have usually been an AJ. Is there a reason the QM should be able to overrule the recommendation from the AJ?

**Comment by scottpeterson** [comment]
The QM will most always be the most experienced and capable official at the table. 

If a QM is flat ignoring their AJ though, that should be handled via other means (like by the Meet Director, or other).

**Comment by kclimenhaga** [comment]
Okay, that makes sense in theory. If it is what happens in practice, then I'm fine with having the rulebook this way. The one time I was at IBQ, I did feel that my QM was ignoring me most of the time, but maybe that was an anomaly.

**Comment by scottpeterson** [comment]
Yeah, that is a bad situation, and unfortunately one that is difficult and uncomfortable to rectify in the moment. And rectifying it in the moment is when it actually matters.

**Comment by ZacharyTinker** [comment]
This is where a system that coaches, quizzers, and other officials can use to help give feedback to the Meet Director during the meet would be beneficial. 

A situation where a QM is not paying attention (intentionally, or absent-mindedly) to the AJ can be drawn to attention by the Meet Director in a quick officials meeting. "Hey everyone, I just want to reiterate that officials need to be conferring with one another. Before this next quiz, be sure to set up some sort of system between yourselves to communicate as needed. For example, a head nod at the end of a quote, etc."

**Comment by jswingle** [comment]
I disagree - the AJ should make the call. QMs are responsible for a great many things, and putting the pressure of difficult rulings off of them is a big plus at higher levels of competition. As a personal anecdote, I've served as an AJ twice at Internationals, and both times was told by the QM that my presence was helpful in distributing the pressure. (Both of those QMs were highly experienced, by the way.)

Within district quizzing at PNW, we often treat the AJ role as a "QM-in-training" thing. That works for us, but I'd prefer we make a district-specific rule to override the rulebook instead of changing the rulebook. The default distribution of roles should be that the QM facilitates a smooth quiz with an uplifting and competitive environment, while the AJ pays special attention to rulings and calls the shots.

From a "best practices" standpoint, officials should always dialogue with each other and present their rulings as unified decisions "from the table."

**Comment by ZacharyTinker** [comment]
My concern with defining final authority to the AJ is for the flow from question to question. If we say AJ has the final authority on all questions, we need to stick to that. 

Now, this is taking it to its farthest extreme, but that would mean the AJ has to approve even the shortest INT questions with no question as to the correctness of the answer. 

I like Scott's idea, that "The Table" has final authority. AJ and QM can't agree, bring in the SK and break the tie.

**Comment by gryphonshafer** [comment]
The meet director can't realistically step in to arbitrate an officials' table impasse every other quiz. Ideally, we want all officials to be well-trained, capable, and work collaboratively. But this is not going to happen all the time. Therefore, there needs to be 1 ultimate authority in the room, and that's going to either be the QM or the AJ. The reality is that in the vast majority of cases, the QM is more experienced than the AJ. And the QM is already ruling unilaterally because the ruling is obvious. Therefore, the QM should be the 1 ultimate authority in the room, with the clear and expressed best practice that the QM work collaboratively with the AJ and SK to derive consensus rulings.

**Comment by jswingle** [comment]
I don't think there are "impasses" every other quiz. They're very rare in practice. I don't know that I ever had a circumstance in which I had to "override" the QM as an AJ.

**Comment by gryphonshafer** [comment]
There were impasses every other quiz in my vague story. They were unfortunately not rare in that case, though I agree normally they are rare normally. Regardless, what's the harm or cost in saying the QM is the ultimate authority in the room but collaboration and consensus should be attempted always?

**Comment by kclimenhaga** [comment]
I agree that either way, there will be situations where there is conflict. To me, the biggest value of saying that the AJ has final say in any debatable rulings is that if the QM has final say, it is easy to just make a ruling without consulting the AJ. If the AJ is supposed to have final say, then there is a stronger push for the QM to consult with the AJ, and if they don't, the AJ can bring it up with the meet director.

I'm fine with stating that the QM has final say if that is the consensus, though my preference is the other way. Regardless, I think the process to hold officials accountable at IBQ should be more firm (doesn't have to be in the rulebook, just in practice).

**Comment by JoshJetto** [comment]
As a quizmaster, I always consult with my answer judge on difficult rulings. I agree with Scott and Gryphon that the quizmaster is almost always the more experienced of the two officials and therefore it makes sense to vest them with the most authority at the table. I utilize the answer judge as a consultant on difficult rulings and in double-checking Quotes and FTVs, and I think that is probably the proper function of the answer judge to the quizmaster. The quizmaster can at his/her discretion choose to delegate the decision-making on any question to the answer judge, but I think in terms of the authority structure that the authority ultimately for making the call should rest with the quizmaster.

**Comment by jttower** [comment]
I agree with the QM as being the final authority, obviously with input from the AJ, for reasons already stated by others.

**Comment by jswingle** [comment]
I think this is a simple enough issue that there's no reason to delay in getting it into the 2023 revision of the rulebook.

It seems that most are in agreement that the rulebook _should_ specify a chain of command between the QM and AJ, but there are different opinions on how the chain of command should go. I lean in the direction of the AJ having the authority, but I think it's more important _that there is a specified authority_ to begin with. I would rather go with specifying the QM than having no specification at all. 

At present, it is ambiguous who actually makes the call, such that there isn't a clear route for resolution when the QM and the AJ disagree. In my own experience I've been in both positions, and 100% of the time have had a good collaborative relationship with the other official even when rulings get tough. But I _have_ observed, as a quizzer, a rather sharp "Paul and Silas" type disagreement which led to the officials just throwing out the question because they couldn't agree. And I think that's silly and something that should never be a part of the quizzing experience. 

tl;dr - Let the adults disagree if they want, but designate someone to make the final call.

**Comment by 24RMiller** [comment]
As discussed in our meeting, I am impartial to this change. However, I could see the benefits of changing the third bullet points under Quizmaster from:
_Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary_
to
_Make decisive rulings, consulting the answer judge when necessary_

I'm not completely sold on the exact wording, but I think clarifying that quizmasters make the ruling but consult the answer judge might give a little more emphasis on the quizmaster's authority without taking anything away from the answer judge.

**Comment by ARMediting** [comment]
I'm in favor of giving the QM the final authority at the table. If good QMs are selected for a tournament, this should not lead to trouble and will give a more confident atmosphere from the table. And it doesn't mean QMs have to rely less on AJs for checking anything or contributing to a ruling.

In addition to a change like the one @24RMiller proposes above, we could add a bullet point to the "quizmaster must" list like "Be the final authority on any disagreements between officials at a table." 

I'd also change the first sentence of 3.2.2.
("When an answer given is in question, the answer judge will determine whether the answer given meets the requirements of a correct answer and make a ruling recommendation to the quizmaster.") to:

> When an answer given is unclear or in question, or an appeal is made, the answer judge will assist the quizmaster in determining the correct ruling, checking all sources of pertinent information if needed to do so. 

This gives the AJ authority and position to ask for the recording to be replayed, look up things in the rulebook, and check the question text against the Scripture.

**Comment by ARMediting** [comment]
If we implement these changes, the opening text of 3.2. may also need changed. Currently, the QM and AJ are given equal weight on deciding to give additional explanations of rulings. Do we want to maintain that? I've typically thought that to be understood as the QM's decision, depending on whether they're expecting a possible appeal. Of course the AJ can be the one to explain a ruling to the quizzers, but it's typically been at the permission or request of the QM.

**Comment by lukeshev** [comment]
I would agree that we need to give authority to one or the other.
My preference would be for the AJ, but good with either.

**Comment by 24RMiller** [comment]
My main concern for giving the AJ the final say is that it could slow down quizzes. Does the qm have to check with the answer judge on every question? Do we use more subjectively terminology such as "unclear or in question" that Alex mentioned earlier? I could potentially get behind the idea of an answer judge making the final ruling on an appeal, but to me, giving the qm authority on the original ruling keeps the quiz process more streamlined.

**Comment by lukeshev** [comment]
Yes, I agree that it has the potential to slow down quizzes. However, I think the benefits of having the AJ as the ruling authority outweigh the extra time it may take.

I would argue that **unless the quizzer's first answer is correct**, the AJ should be consulted to determine if the quizzer gave an incorrect / out of context response.

The QM's job is to facilitate the quiz - to announce/ask questions, give prompts, announce rulings, timeouts, etc.
The AJ's job is to rule. Having AJs as the final authority on rulings makes sense because AJs are able to focus _completely_ on that one job, whereas QMs must be focused on the question on the card and prompt the quizzer where necessary.

The AJ also has access to more resources than a QM does (material, concordance, rule book).

If the quizzer gives a correct answer the first time, the QM should rule immediately on that answer as there is no question about it. The quiz can move on uninterrupted.
If the QM is unsure where a quizzer is quoting, or if the quizzer did not give a correct answer the first time, they should consult their AJ, who will give a ruling - which can be as short as a nod or shake of the head. The QM/AJ should communicate with each other to make an accurate ruling.

I believe that unless the quizzer's first answer is correct, it is always good to do a quick double check with the AJ. This check can also help prevent appeals, which would shorten the length of the quiz.
Even if the quizzer's first answer is correct, I think it is a good idea to double check- especially on Q/F type questions, as they need to be word perfect.

To avoid a conflict of interest, the AJ should be the final authority for appeals, as a QM's actions are grounds for an appeal.

Again, I'm good with either, but in my mind, the benefits of having the AJ be the authority greatly outweigh the extra time it may take for them to discuss their ruling with the QM.
